model_configs/multilabel_classification/datasets.jsonnet

```diff
@@ -26,4 +26,81 @@
     validation_file: 'val.arff',
     test_file: 'test.arff',
   },
-}
```

Comment:
These changes won't affect --> the same with the 'feat/new-dataset' branch (which will be merged to the main branch). Should have excluded these file in the commit. Apologies for confusion.